style.css

```diff
@@ -0,0 +1,3 @@
+body{
+  backround-color:red
```

Comment:
please always remember to terminate after every line of code in your css , your editor will not recognize the code and it will give errors if the line is not terminated.